ext/snmp/snmp.stub.php

```diff
@@ -75,6 +75,22 @@ function snmp_read_mib(string $filename): bool {}
 
 class SNMP
 {
+    public array $info;
+    /** @var int|null */
+    public $max_oids;
```

Comment:
Hm, I'm not sure about declaring these non-readonly properties, due to the null default values :/

Reply:
Fortunately, it's possible to declare a default value for `$oid_increasing_check`, `$exceptions_enabled` (`true` and `0`, respectively). The others are a question, yes. :S My preference would be to declare them as `nullable`, so that at least we provide complete property reflection info (even though PHPDoc-based types in the stubs are not completely right).

Reply:
I'd also be ok with going with a pragmatic approach: document the intended type (not considering the implicit null default value). I think this is similar what did with the parameters in a few cases - so that at least we can have meaningful documentation.

If you also think it makes sense, I'll conduct an RFC where the internal property handling could be made more consistent (by converting them to typed properties and by making write handlers respect the strict_types directive). What do you think? Is it possible to do in a minor release?

Reply:
Personally, I'd be happy to have these respect property types / strict types in 8.1 without an RFC. I doesn't seem like a big deal to me.

Reply:
Sounds good!

Reply:
I've just realized that I should make these properties typed. I forgot about it, but I'll do it tomorrow (maybe)